modules/tinymce/src/themes/silver/main/ts/Events.ts

```diff
@@ -42,10 +50,15 @@ const setup = (editor: Editor, mothership: Gui.GuiSystem, uiMothership: Gui.GuiS
   });
 
   // Editor content events
-  const onContentClick = (raw: UIEvent) => broadcastOn(Channels.dismissPopups(), { target: SugarElement.fromDom(raw.target as Node) });
+  const onContentClick = () => {
```

Comment:
This logic needs to be on `mousedown`, not on `mouseup`/`click`. If you recall when we were discussing this, I showed that we'd need a new `onContentMousedown` function to handle that case.

- [ ] Revert this and add a new `onContentMousedown` handle please